### DISCUSSION TITLE
Is IT industry Job Available?

### DISCUSSION THREAD

**Discussion by DivekarNikhil** [community/community#112935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Many company  fire employee dat by day and I am currently pursuing my BE Degree in Computer Engineering . How i get job in IT field because all ready Experiance  employee fire company and I am inter for job is possible to get job

**Comment by queenofcorgis** [comment]
Hi @DivekarNikhil , we're happy you're here 😀

However, we currently do not allow job listings or seeking on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
